Lib/test/libregrtest/runtest.py

```diff
@@ -327,7 +327,7 @@ def cleanup_test_droppings(test_name, verbose):
                                f"directory nor file")
 
         if verbose:
-            print_warning("%r left behind %s %r" % (test_name, kind, name))
+            print_warning(f"{test_name} left behind {kind} {name!r}")
```

Comment:
```suggestion
            print_warning(f"{test_name!r} left behind {kind} {name!r}")
```

The original code looks like this, is this change intended?

Reply:
I think that it was a bug in the old code: see the print_warning() below (line 339).

Reply:
Got it :)